spyder_kernels/customize/umr.py

```diff
@@ -92,7 +92,7 @@ def run(self):
         # Report reloaded modules
         if self.verbose and modnames_to_reload:
             modnames = modnames_to_reload
-            print("\x1b[4;33m%s\x1b[24m%s\x1b[0m"
+            print("\x1b[1;4;31m%s\x1b[24m%s\x1b[0m"
```

Comment:
In order to make this color configurable according to the light/dark theme, you need to make `UserModuleReloader` receive `shell` as a kwarg, then pass `self.shell` to it here:

https://github.com/spyder-ide/spyder-kernels/blob/4e554db2ccf2bafa30a6dd2c5c74213e013efa0c/spyder_kernels/customize/code_runner.py#L148-L150

and finally add two methods to `SpyderShell` called `set_spyder_theme`/`get_spyder_theme` to set and get the current theme in a private variable. You'll need to call `set_spyder_theme` from `SpyderKernel` (when you get the theme info from Spyder) and `get_spyder_theme` here to adjust the color.